### ISSUE TITLE
`bun install` error

### ISSUE THREAD

**Issue by raziman18** [oven-sh/bun#410]
I faced an error when trying to run `bun install` .

Error log:
```
[0.02ms] ".env"
bun install v0.1.1
  🚚 common-tags [177/397]
SegmentationFault at 188


–––– bun meta ––––
Bun v0.1.1 Linux x64 #1 SMP PREEMPT_DYNAMIC Wed, 22 Jun 2022 19:52:19 +0000
InstallCommand: dotenv
HTTP: 157
Elapsed: 20823ms | User: 20748ms | Sys: 165ms
RSS: 0.30GB | Peak: 0.23GB | Commit: 0.30GB | Faults: 42
–––– bun meta ––––

Ask for #help in https://bun.sh/discord or go to https://bun.sh/issues
```

I will also give you the repository that I used that produced the error:
https://github.com/issadarkthing/dungeon-rpg

**Comment by FinnRG** [comment]
Could you please run ```bun install --verbose 2> logs.txt``` and  ```sudo perf trace bun install --silent > log.perf.txt 2>&1``` and post the output?

**Comment by raziman18** [comment]
`sudo perf trace bun install --silent > log.perf.txt 2>&1` [log.perf.txt](https://github.com/Jarred-Sumner/bun/files/9072648/log.perf.txt)
`bun install --verbose 2> logs.txt` [logs.txt](https://github.com/Jarred-Sumner/bun/files/9072649/logs.txt)

**Comment by FinnRG** [comment]
@issadarkthing  I'm not able to reproduce this issue. Could you please run ```bun upgrade``` and try it again with v0.1.2?

**Comment by raziman18** [comment]
Same error:
```
[0.02ms] ".env"
bun install v0.1.2
  🚚 ts-node [350/618]
SegmentationFault at 188


–––– bun meta ––––
Bun v0.1.2 Linux x64 #1 SMP PREEMPT_DYNAMIC Wed, 22 Jun 2022 19:52:19 +0000
InstallCommand: dotenv
HTTP: 263
Elapsed: 35670ms | User: 35871ms | Sys: 214ms
RSS: 0.30GB | Peak: 0.20GB | Commit: 0.30GB | Faults: 0
–––– bun meta ––––

Ask for #help in https://bun.sh/discord or go to https://bun.sh/issues
```

**Comment by isharaman8** [comment]
It can be fixed with `bun install --backend=copyfile
`
https://github.com/Jarred-Sumner/bun/issues/359

**Comment by TKasperczyk** [comment]
I had the same issue, which was solved by disconnecting my OpenVPN tunnel and making sure that the default gateway isn't a proxy. Otherwise it kept throwing segfaults at random times during the installation process. It's worth noting that `bun install` **always** crashes at some point when I'm connected through a VPN, and **always** works when I'm connected to the internet directly.

**Comment by Jarred-Sumner** [comment]
Should be fixed as of Bun v0.1.12 but if not please comment and I'll re-open

@TKasperczyk I'd especially appreciate if you could try again with your OpenVPN tunnel. I haven't tested the changes on a VPN yet.
